src/commands/cmd_json.cc

```diff
@@ -624,6 +632,37 @@ class CommandJsonMSet : public Commander {
   }
 };
 
+class CommandJsonDebug : public Commander {
+ public:
+  Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    redis::Json json(svr->storage, conn->GetNamespace());
+
+    std::string path = "$";
+
+    if (!util::EqualICase(args_[1], "memory")) {
+      return {Status::RedisExecErr, "ERR wrong number of arguments for 'json.debug' command"};
+    }
+
+    if (args_.size() == 4) {
+      path = args_[3];
+    } else if (args_.size() > 4) {
+      return {Status::RedisExecErr, "The number of arguments is more than expected"};
+    }
+
+    std::vector<std::size_t> results;
+    auto s = json.DebugMemory(args_[2], path, &results);
+
+    if (s.IsNotFound()) {
+      *output = conn->NilString();
```

Comment:
It's wrong. For nonexistent key, the result is `0`.

Please confirm with redis-stack and redis-cli by yourself.

Reply:
Thank you again. The latest version of Redis Stack returns all cases, and I will make the necessary modifications later
![image](https://github.com/apache/kvrocks/assets/127465317/565cb27d-3b80-4757-838f-7a1dc69a779f)